Cubical/Data/Nat/Order.agda

```diff
@@ -92,6 +98,9 @@ pred-≤-pred (k , p) = k , injSuc ((sym (+-suc k _)) ∙ p)
   l3 : 0 ≡ i
   l3 = sym (snd (m+n≡0→m≡0×n≡0 l2))
 
+≤-+ : m ≤ n → l ≤ k → m + l ≤ n + k
+≤-+ p q = ≤-trans (≤-+k p) (≤-k+ q)
+
```

Comment:
How about renaming this to `≤-+-≤` to match `<-+-≤`?

Reply:
done